airbyte-integrations/connectors/source-box-data-extract/metadata.yaml

```diff
@@ -0,0 +1,39 @@
+data:
+  ab_internal:
+    ql: 300
+    sl: 300
+  allowedHosts:
+    hosts:
+      - box.com
+      - boxcloud.com
+  # registries:
+  #   oss:
+  #     enabled: true
+  #   cloud:
+  #     enabled: false
+  remoteRegistries:
+    pypi:
+      enabled: true
+      packageName: airbyte-source-box-data-extract
+  connectorBuildOptions:
+    # Please update to the latest version of the connector base image.
+    # https://hub.docker.com/r/airbyte/python-connector-base
+    # Please use the full address with sha256 hash to guarantee build reproducibility.
+    baseImage: docker.io/airbyte/python-connector-base:3.0.0@sha256:1a0845ff2b30eafa793c6eee4e8f4283c2e52e1bbd44eed6cb9e9abd5d34d844
```

Comment:
I'd suggest using 4.0.0 (lookup the hash on dockerbase)

Reply:
Done